index.html

```diff
@@ -1,279 +1,274 @@
+                        </div>
+                    </div>
+                </div> -->
+    
+               <!-- <div class="recent_work">
+                   <div class="second_project first_project">
+                        <div class="first_project_holder">
+                            <h3 class="first_project_title">Data Dashboard Healthcare</h3>
+                            <p class="first_project_content">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                            <ul class="first_project_skills">
+                                <li><p class="first_project_holder1">html</p>
+                                </li>
+                                <li><p class="first_project_holder1">bootstrap</p>
+                                </li>
+                                <li><p class="first_project_holder1">Ruby</p>
+                                </li>
+                            </ul>
+                             <button class="see_project_button2 see_project_button2a">See Project</button>
+                        </div>
+                    </div>
+                </div>-->
+    
+                <div class="recent_work">
+                    <div class="third_project first_project">
+                        
+                        <div class="first_project_holder2">
+                            <h3 class="first_project_title">Website Protfolio</h3>
+                            <p class="first_project_content">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                            <ul class="first_project_skills">
+                                <li><p class="first_project_holder1">html</p>
+                                </li>
+                                <li><p class="first_project_holder1">bootstrap</p>
+                                </li>
+                                <li><p class="first_project_holder1">Ruby</p>
+                                </li>
+                            </ul>
+                             <button class="see_project_button2 see_project_button2a">See Project</button>
+                        </div>
+                    
+                </div>
+            </div>
+        </div>
     </section>
-    <section class="third_section_container">
-      <div class="about_us_div">
-          <h2 class="about_title">About me</h2>
-          <p class="about_content">
-            Hello I’m a software developer! I can help you build a product ,
-            feature or website Look through some of my work and experience! If
-            you like what you see and have a project you need coded, don’t
-            hestiate to contact me.
-          </p>
-           <div class="resume_button">
-       <a class="button_resume"> Get My Resume </a>
```

Comment:
- [ ] Please follow the Figma design as much as possible. it will sharpen your skills for your further career. Kindly add the background for this section.
## your design👇
![skillsst](https://user-images.githubusercontent.com/66992901/194668881-589e2e11-ea98-4c5e-a936-8bbe2327b236.PNG)
## Figma design👇
![skillsfigma](https://user-images.githubusercontent.com/66992901/194668912-cfe95af9-4316-41c2-bd13-ba8183b96e4d.PNG)